5calls/app/src/main/assets/licenses.html

```diff
@@ -48,6 +48,18 @@
         <p>Copyright 2016</p>
         <p>Jonas Gehring</p>
     </li>
+    <li>
+        <b>OkHttp3</b>
+        <br/>
+        <p>Copyright 2017</p>
+        <p>Square Inc.</p>
+    </li>
+    <li>
+        <b>OkHttpStack.java</b>
+        <br/>
+        <p>Copyright 2016</p>
+        <p>Eric Cochran</p>
+    </li>
```

Comment:
Please also add GSON under the apache license: https://github.com/google/gson/blob/master/LICENSE

Reply:
Already there: https://github.com/bryansills/5calls-android/blob/5df680a0852868928f4e6a9d6a09d67014ed599b/5calls/app/src/main/assets/licenses.html#L28